py3_analysis/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM socrata/python3
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y python-numpy python-scipy
+
+RUN pip install scikit-learn==0.17
```

Comment:
Odd that we don't do it in the python2 version. I also notice that in the clads dockerfile, we're doing that, then pip installing numpy and scipy instead of apt-get installing. Perhaps that's why we needed those installs there, and if we apt-get them instead, we don't need those build tools?